frontend/app/src/components/v1/molecules/combobox/combobox.tsx

```diff
@@ -164,7 +164,7 @@ export function Combobox({
           )}
         </Button>
       </PopoverTrigger>
-      <PopoverContent className="w-[300px] p-2" align="start">
+      <PopoverContent className="w-[300px] p-2 z-[70]" align="start">
```

Comment:
this was a bug - it was causing the filters to not pop up on top of the side sheet